packages/slate/src/models/point.js

```diff
@@ -120,6 +120,79 @@ class Point extends Record(DEFAULTS) {
     return !this.isSet
   }
 
+  /**
+   * Check whether the point is after a `range`.
+   *
+   * @return {Boolean}
+   */
+
+  isAfterRange(range) {
```

Comment:
Could you refactor this to add `isAfterPoint(point)` which can then be used inside `isAfterRange(range)`? That way we get both options.

Reply:
Sounds good, I'll fix it.